bin/install-as-user

```diff
@@ -92,19 +94,23 @@ then
 fi
 echo $DONE_MSG
 
-# Create the database if it doesn't exist:
-echo -n "Setting up database... "
-if ! psql -l | egrep "^ *$DB_NAME *\|" > /dev/null
-then
-    createdb --owner "$UNIX_USER" "$DB_NAME"
-    echo 'CREATE LANGUAGE plpgsql;' | psql -U "$UNIX_USER" "$DB_NAME" || true
+if [ $INSTALL_DB = true ]; then
+    # Create the database if it doesn't exist:
+    echo -n "Setting up database... "
+    if ! psql -l | egrep "^ *$DB_NAME *\|" > /dev/null
+    then
+        createdb --owner "$UNIX_USER" "$DB_NAME"
+        echo 'CREATE LANGUAGE plpgsql;' | psql -U "$UNIX_USER" "$DB_NAME" || true
+    fi
+    bin/update-schema --commit
+    echo $DONE_MSG
```

Comment:
It does seem (once update-all-reports moved up here) that this overlaps with the middle bit of docker.preinit but not sure if worth factoring together somehow or not.

Reply:
Not sure I completely understand - they run at such different stages in the lifecycle? Or do you mean calling out from the preinit script in some way to avoid the repetition?

It might be worth checking in some way for `plpgsql`, this just generates an error on more recent versions and it feels a bit inelegant (I left it out of the preinit script altogether).

Reply:
Yeah, just the avoiding repetition was all, no biggie.